### PR TITLE
(chore) simplify swift

### PR DESCRIPTION
**Pull request by joshgoebel**
### Changes

Take advantage of new features to simplify swift

### Checklist
- [x] Added markup tests, or they don't apply here because...
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by joshgoebel** [comment]
@svanimpe Any chance you could review this?  Simplies swift using all the new stuff... all tests pass - the only changes to tests are to deal with the function => title.function updates...

File: src/languages/swift.js [hunk 1]
```diff
@@ -399,11 +391,12 @@ export default function(hljs) {
...
-    match: /\b(subscript|init[?!]?)\s*(?=[<(])/,
-    keywords: {
-      keyword: "subscript init init? init!",
-      $pattern: /\w+[?!]?/
+    match: [
+      /\b(?:subscript|init[?!]?)/,
+      /\s*(?=[<(])/,
+    ],
+    className: {
+      1: "keyword"
```
Comment (joshgoebel) [inline src/languages/swift.js:399]: I wonder if this shouldn't be `title.function` instead of keyword?

**Review by NullVoxPopuli — APPROVED** [review]
I don't know swift, but this looks like a fairly straightforward change

File: src/languages/swift.js (hunk 1 above)
Comment (svanimpe) [reply src/languages/swift.js:399]: If you mean `subscript` and `init`, those are just keywords. Subscripts and initializers are type members that don't have a title.

**Pull request merged**